### PR TITLE
Release v1.0.6

### PR DESCRIPTION
**Pull request by shreyas-badiger**
## [v1.0.5] - 2022-06-02
* c45365b multi arch support

### CODE REVIEW

**Review by sahilbadla — APPROVED** [review]
LGTM

**Comment by shreyas-badiger** [comment]
DON NOT MERGE until multi-arch PR is merged.

**Pull request merged**
